lib/internal/http2/core.js

```diff
@@ -1250,6 +1252,25 @@ class Http2Session extends EventEmitter {
     this[kHandle].setNextStreamID(id);
   }
 
+  // Sets the local window size (local endpoints's window size)
+  // Returns 0 if sucess or throw an exception if NGHTTP2_ERR_NOMEM
+  // if the window allocation fails
+  setLocalWindowSize(windowSize) {
+    if (this.destroyed)
+      throw new ERR_HTTP2_INVALID_SESSION();
+
+    validateNumber(windowSize, 'windowSize');
+    if (windowSize <= 0 || windowSize > kMaxInt)
+      throw new ERR_OUT_OF_RANGE('windowSize', `> 0 and <= ${kMaxInt}`,
+                                 windowSize);
```

Comment:
Ref: [nghttp2_session_set_local_window_size](https://nghttp2.org/documentation/nghttp2_session_set_local_window_size.html#c.nghttp2_session_set_local_window_size)

Maybe we should with `validateInteger`, since `window_size` is `int32`.

Reply:
Landed, use `validateInt32` here.